lte/gateway/python/magma/pipelined/ifaces.py

```diff
@@ -31,8 +32,31 @@ def monitor_ifaces(iface_names):
         yield from asyncio.sleep(POLL_INTERVAL_SECONDS)
 
 
-def get_mac_address(interface_name: str) -> str:
-    if_addresses = netifaces.ifaddresses(interface_name)[netifaces.AF_LINK]
-    if not if_addresses or not if_addresses[0].get('addr'):
-        raise ValueError(f"No mac address found for interface {interface_name}")
-    return if_addresses[0]['addr']
+def get_mac_address_from_iface(iface: str) -> str:
+    ifaddress = netifaces.ifaddresses(iface)
+    if netifaces.AF_LINK in ifaddress:
```

Comment:
The old code and the new code have different kinds of null checks. Do we know which of these cases can actually happen with the library? If not, maybe we should do all of them.
(Just for consistency, it's not that important since we're going to raise an exception in any case.)